main.js

```diff
@@ -1,104 +0,0 @@
-let Attributes = document.getElementsByTagName("tbody"),
-  count = document.getElementsByTagName("p"),
-  input = document.getElementById("input"),
-  addBtn = document.getElementById("addBtn"),
-  Search = document.getElementById("search"),
-  Todos,
-  Temp = [],
-  inpEdit = "";
-
-function fetchData() {
-  Todos = JSON.parse(localStorage.getItem("todos"));
-  Temp = Todos;
-  ShowTodo();
-}
-fetchData();
-function ShowTodo() {
```

Comment:
please write the functions name in `camelCase`

Reply:
Okay, I'll edit it